test/samples/json/static/figure.json

```diff
@@ -0,0 +1,303 @@
+            "body": {
+              "serial": "container_containerstatic_6296313b7a3e8597398679",
+              "body": "choice #2",
+              "elements": {},
+              "attributes": [],
+              "debug": {
+                "relatedItem": "item_6296313b6548e386874236"
+              }
+            },
+            "debug": {
+              "relatedItem": "item_6296313b6548e386874236"
+            }
+          },
+          "choice_simplechoice_6296313b7ad7b956475953": {
+            "identifier": "choice_3",
+            "serial": "choice_simplechoice_6296313b7ad7b956475953",
+            "qtiClass": "simpleChoice",
+            "attributes": {
+              "identifier": "choice_3",
+              "fixed": false,
+              "showHide": "show"
+            },
+            "body": {
+              "serial": "container_containerstatic_6296313b7b3a2970805940",
+              "body": "choice #3",
+              "elements": {},
+              "attributes": [],
+              "debug": {
+                "relatedItem": "item_6296313b6548e386874236"
+              }
+            },
+            "debug": {
+              "relatedItem": "item_6296313b6548e386874236"
+            }
+          }
+        },
+        "prompt": {
+          "serial": "container_containerstatic_6296313b727b4692175752",
+          "body": "test figure\n{{figure_6296313b72cee277084465}}\n<span>\u00a0<\/span>",
+          "elements": {
+            "figure_6296313b72cee277084465": {
+              "serial": "figure_6296313b72cee277084465",
+              "qtiClass": "figure",
+              "attributes": {},
+              "body": {
+                "serial": "container_containerfigure_6296313b7326f634438308",
+                "body": "{{img_6296313b7402f205167065}}{{figcaption_6296313b75ce7386161516}}",
+                "elements": {
+                  "img_6296313b7402f205167065": {
+                    "serial": "img_6296313b7402f205167065",
+                    "qtiClass": "img",
+                    "attributes": {
+                      "src": "tao-logo.png",
+                      "alt": "tao logo",
+                      "width": "20%",
+                      "type": "image\/png"
+                    },
+                    "debug": {
+                      "relatedItem": "item_6296313b6548e386874236"
+                    }
+                  },
+                  "figcaption_6296313b75ce7386161516": {
+                    "serial": "figcaption_6296313b75ce7386161516",
+                    "qtiClass": "figcaption",
+                    "attributes": {},
+                    "body": {
+                      "serial": "container_containerfigcaption_6296313b75f0f866849296",
+                      "body": "test caption",
+                      "elements": {},
+                      "attributes": [],
+                      "debug": {
+                        "relatedItem": "item_6296313b6548e386874236"
+                      }
+                    },
```

Comment:
Note: in Solar we ended up with a slightly different JSON model. Only the `figure` and the `img` are their own elements, and the `figcaption` remains as static HTML string in the `figure.body` (we decided there was nothing to gain from deconstructing this element).

It may be inconsistent, but I think this is ok, since Terre and Solar have their own renderers and JSON providers... right?

Reply:
Yes, it can be done different. On Terra `figcaption` is element